src/drivers/net/virtio_mmio.rs

```diff
@@ -166,7 +166,7 @@ impl VirtioNetDriver {
 			match drv.init_dev() {
 				Err(error_code) => Err(VirtioError::NetDriver(error_code)),
 				_ => {
-					//drv.print_information();
+					drv.print_information();
```

Comment:
`print_information` can most probably removed with a debug format print. But this doesn't look like dead code removal to me

Reply:
While technically not removing dead code, it removes the dead code warning of `print_information` and `dev_status`, by making that code live.

Those functions themselves intentionally print on the `info` level, which seems a good choice to me, similar to the other info reports on detected devices on boot.